src/test/java/org/opentripplanner/raptor/moduletests/F04_AccessEgressWithRidesNoTransitTest.java

```diff
@@ -0,0 +1,113 @@
+import static org.opentripplanner.raptor._data.api.PathUtils.pathsToString;
+import static org.opentripplanner.raptor._data.transit.TestAccessEgress.flex;
+import static org.opentripplanner.raptor._data.transit.TestRoute.route;
+import static org.opentripplanner.raptor._data.transit.TestTransfer.transfer;
+import static org.opentripplanner.raptor._data.transit.TestTripPattern.pattern;
+import static org.opentripplanner.raptor._data.transit.TestTripSchedule.schedule;
+import static org.opentripplanner.raptor.moduletests.support.RaptorModuleTestConfig.TC_MIN_DURATION;
+import static org.opentripplanner.raptor.moduletests.support.RaptorModuleTestConfig.TC_MIN_DURATION_REV;
+import static org.opentripplanner.raptor.moduletests.support.RaptorModuleTestConfig.multiCriteria;
+import static org.opentripplanner.raptor.moduletests.support.RaptorModuleTestConfig.standard;
+
+import java.util.List;
+import javax.annotation.Nonnull;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.opentripplanner.raptor.RaptorService;
+import org.opentripplanner.raptor._data.RaptorTestConstants;
+import org.opentripplanner.raptor._data.api.PathUtils;
+import org.opentripplanner.raptor._data.transit.TestTransitData;
+import org.opentripplanner.raptor._data.transit.TestTripSchedule;
+import org.opentripplanner.raptor.api.request.RaptorRequestBuilder;
+import org.opentripplanner.raptor.configure.RaptorConfig;
+import org.opentripplanner.raptor.moduletests.support.RaptorModuleTestCase;
+import org.opentripplanner.raptor.spi.UnknownPathString;
+
+/**
+ * FEATURE UNDER TEST
+ * <p>
+ * Raptor should be able to timeshift a Flex ~ Walk ~ Flex path.
+ */
+public class F04_AccessEgressWithRidesNoTransitTest implements RaptorTestConstants {
+
+  private final TestTransitData data = new TestTransitData();
+  private final RaptorService<TestTripSchedule> raptorService = new RaptorService<>(
+    RaptorConfig.defaultConfigForTest()
+  );
+
+  @BeforeEach
+  public void setup() {
+    data
+      .withRoute(
+        // Pattern arrive at stop 2 at 0:03:00
+        route(pattern("R1", STOP_A, STOP_D))
+          .withTimetable(
+            // First trip is too early: It takes 2m to get to the point of boarding:
+            // --> 00:00:00 + flex 30s + slack(1m + 30s) = 00:02:00
+            schedule().departures("0:03:29, 0:05:29"),
+            // This is the trip we expect to board
+            schedule().departures("0:04:00, 0:10:00").arrivals("0, 00:06:00"),
+            // REVERSE SEARCH: The last trip arrives too late: It takes 1m40s to get to the
+            // point of "boarding" in the reverse search:
+            // --> 00:10:00 - (flex 20s + slack(1m + 10s)) = 00:08:30  (arrival time)
+            schedule().arrivals("0:04:51, 0:06:51")
+          )
+      )
+      .withTransfer(RaptorTestConstants.STOP_B, transfer(STOP_C, D5m));
```

Comment:
```suggestion
      .withTransfer(STOP_B, transfer(STOP_C, D5m));
```

Reply:
You implement the interface where these constants are defined, no need to prefix them.